### PR TITLE
Update JPA 3.0 API to RC2 and Eclipselink 3.0 to 879de6b.

### PR DESCRIPTION
**Pull request by jgrassel**
Update the Jakarta Persistence APIs to Release Candidate 2 and Eclipselink 3.0 to master-879de6b.  Add com.ibm.ws.jpa_30_fat to test using v3.0 persistence.xml schema.

### CODE REVIEW

**Comment by jgrassel** [comment]
#build

**Comment by jgrassel** [comment]
Failing test looks unrelated to JPA.

File: dev/com.ibm.ws.jpa_30_fat/build.gradle [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
```
Comment (dazey3) [inline dev/com.ibm.ws.jpa_30_fat/build.gradle:2]: Headers update to 2020

File: dev/com.ibm.ws.jpa_30_fat/test-applications/jpabootstrap/resources/jpa-3.0/WEB-INF/classes/META-INF/persistence.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<persistence xmlns="https://jakarta.ee/xml/ns/persistence"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://jakarta.ee/xml/ns/persistence https://jakarta.ee/xml/ns/persistence/persistence_3_0.xsd"
+        version="3.0">
+    <persistence-unit name="JPAPU">
+    	<properties>
+			<!-- EclipseLink should create the database schema automatically -->
+			<property name="eclipselink.ddl-generation" value="drop-and-create-tables" />
```
Comment (dazey3) [inline dev/com.ibm.ws.jpa_30_fat/test-applications/jpabootstrap/resources/jpa-3.0/WEB-INF/classes/META-INF/persistence.xml:8]: Don't we use DDL generation? If this is necessary, can we change to: 
```
<property name="jakarta.persistence.schema-generation.database.action" value="drop-and-create"/>
```

File: dev/com.ibm.ws.jpa_30_fat/bnd.bnd [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	test-applications/helpers/DatabaseManagement/src,\
+	test-applications/jpabootstrap/src,\
+	test-framework/src
+
+fat.project: true
+tested.features: el-3.0, jpa-3.0, servlet-5.0, cdi-3.0, beanValidation-3.0, enterprisebeanslite-4.0
+
+# Dependencies may be local bundles (e.g. com.ibm.websphere.javaee.servlet.3.1)
+#      or binaries from Artifactory (e.g. commons-logging:commons-logging)
+-buildpath: \
```
[the remark this answers is not in the document]
Comment (jgrassel) [reply dev/com.ibm.ws.jpa_30_fat/bnd.bnd:26]: When #14235 is delivered, I can link this to the common jpa test framework project and get rid of them entirely.

File: dev/com.ibm.ws.jpa_30_fat/test-applications/jpabootstrap/resources/jpa-3.0/WEB-INF/classes/META-INF/persistence.xml (hunk 2 above)
Comment (jgrassel) [reply dev/com.ibm.ws.jpa_30_fat/test-applications/jpabootstrap/resources/jpa-3.0/WEB-INF/classes/META-INF/persistence.xml:8]: Don't think we have that set up for this test, a todo for the future.

File: dev/com.ibm.ws.jpa_30_fat/build.gradle (hunk 1 above)
Comment (jgrassel) [reply dev/com.ibm.ws.jpa_30_fat/build.gradle]: Fixed.

File: dev/com.ibm.ws.jpa_30_fat/test-framework/src/com/ibm/ws/testtooling/vehicle/web/JPATestServlet.java [hunk 4]
```diff
@@ -0,0 +1,156 @@
+/**
```
Comment (tkburroughs) [inline dev/com.ibm.ws.jpa_30_fat/test-framework/src/com/ibm/ws/testtooling/vehicle/web/JPATestServlet.java:1]: Copyright is missing

**Review by tkburroughs — COMMENTED** [review]
Noticed one file without any copyright; otherwise looks good to me.

**Review by dazey3 — APPROVED** [review]
guess we'll clean it up later

File: dev/com.ibm.ws.jpa_30_fat/test-framework/src/com/ibm/ws/testtooling/vehicle/web/JPATestServlet.java (hunk 4 above)
Comment (jgrassel) [reply dev/com.ibm.ws.jpa_30_fat/test-framework/src/com/ibm/ws/testtooling/vehicle/web/JPATestServlet.java]: Thanks for catching that. Fixed!

**Comment by jgrassel** [comment]
#run-libby-bot

**Review by tkburroughs — APPROVED** [review]

**Comment by jgrassel** [comment]
Test failures are UI Fat, not related to JPA.

**Pull request merged**